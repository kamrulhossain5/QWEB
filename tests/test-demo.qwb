function int gcd(int a, int b){
  REPEAT until (a != b){
    if (a > b) a = a - b;
    otherwise b = b - a;
  }
  output a;
}

function int main(){
  createHeader("Demo Program");
  createSubheader("Greatest Common Divisor");
  createParagraph("The gcd function returns the greatest common divisor of two or more integers. The greatest common divisor is the largest integer that goes into all supplied numbers without a remainder. For example, gcd(60,36) returns 12.");
  createImage("./demo.png");
  createList("gcd(2,14)");
  print(gcd(2,14));
  createList("gcd(3,15)");
  print(gcd(3,15));
  createList("gcd(99,121)");
  print(gcd(99,121));
  output 0;
}