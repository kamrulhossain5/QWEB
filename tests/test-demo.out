<h1>Demo Program</h1>
<h2>Greatest Common Divisor</h2>
<p>The gcd function returns the greatest common divisor of two or more integers. The greatest common divisor is the largest integer that goes into all supplied numbers without a remainder. For example, gcd(60,36) returns 12.</p>
<img src='./demo.png'>
<li>gcd(2,14)</li>
2<br>
<li>gcd(3,15)</li>
3<br>
<li>gcd(99,121)</li>
11<br>
